tests/test_token_revocation.py

```diff
@@ -52,13 +52,13 @@ def test_revoke_access_token(self):
             expires=timezone.now() + datetime.timedelta(days=1),
             scope="read write"
         )
-        query_string = urlencode({
+        payload = {
             "client_id": self.application.client_id,
             "client_secret": self.application.client_secret,
             "token": tok.token,
-        })
-        url = "{url}?{qs}".format(url=reverse("oauth2_provider:revoke-token"), qs=query_string)
-        response = self.client.post(url)
+        }
+        url = "{url}".format(url=reverse("oauth2_provider:revoke-token"))
```

Comment:
This is rather long. Why not just `url = reverse("oauth2_provider:revoke-token"`?